ipatests/test_xmlrpc/test_pwpolicy_plugin.py

```diff
@@ -31,6 +31,49 @@
                                               Declarative)
 
 
+def pwpolicy_cmd(
+        cmd, minlife, cospriority=None,
+        pwpolicy_group=None, pwpolicyadd=False):
+    if pwpolicyadd:
```

Comment:
Add a method docstring: either create a new pwpolicy with max life = 1 day or modify the existing global pwpolicy.